classification_analysis.ipynb

```diff
@@ -240,29 +323,57 @@
     "                                                                                  tr['start_ts'],\n",
     "                                                                                  tr['end_ts'])\n",
     "                        \n",
-    "                        if section_gt_leg[\"type\"] == \"WAITING\": \n",
+    "                        if section_gt_leg[\"type\"] == \"WAITING\":\n",
```

Comment:
> "How are we expected to infer the modes of each trip for the reference GT trajectory?"

`section_gt_leg`. the ground truth for each section - e.g. 
https://github.com/MobilityNet/mobilitynet-analysis-scripts/blob/eac11f8356dae7825257a40495183f846077d746/spec_creation/final_sfbayarea_filled_reroutes/unimodal_trip_car_bike_mtv_la.filled.reroute.json#L230

OR

https://github.com/MobilityNet/mobilitynet-analysis-scripts/blob/eac11f8356dae7825257a40495183f846077d746/bin/data/shankari%40eecs.berkeley.edu/unimodal_trip_car_bike_mtv_la/config~evaluation_spec/0_9223372036854775807.json#L241

(they are essentially copies of each other)

Just like you get the type from the ground truth, you can get the mode